dependency-check-suppressions.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<suppressions xmlns="https://jeremylong.github.io/DependencyCheck/dependency-suppression.1.1.xsd">
+  <suppress>
```

Comment:
Should we put a comment explaining the reason for each suppression?

Reply:
I think that’s a good idea.

Reply:
https://github.com/cerner/beadledom/pull/80/commits/cf6b74240c7390911c6ba7c4d8489d593d9e78ac